pkg/app/app.go

```diff
@@ -286,7 +286,7 @@ func (c *App) OAuth2Callback(w http.ResponseWriter, r *http.Request) error {
 	if err != nil {
 		return err
 	}
-	tk, err := c.oauth2Helper.Exchange(context.Background(), authCode)
+	tk, err := c.oauth2Helper.OAuthConfig.Exchange(context.Background(), authCode)
```

Comment:
Ditto, I think we should care when `helper.OAuthConfig` is nil. Unless we have a check on OAuthConfig, we would face at unexpected behavior.